application/src/messengercore.h

```diff
@@ -0,0 +1,29 @@
+#ifndef MESSENGERCORE_H
+#define MESSENGERCORE_H
+
+#include <QObject>
+#include <QByteArray>
+#include <QTcpSocket>
+
+class MessengerCore : public QObject
+{
+    Q_OBJECT
+public:
+    explicit MessengerCore(QObject *parent = nullptr);
+
+    void sendToServer(const QByteArray data);
```

Comment:
```suggestion
    void sendToServer(const QByteArray &data);
```
зачем целый массив копировать? передавай по ссылке